bedrock/firefox/templates/firefox/whatsnew_42/learnmore.html

```diff
@@ -0,0 +1,35 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew_42/base.html" %}
+
+{% add_lang_files "firefox/whatsnew_42" %}
+
+{% block body_class %}firefox-up-to-date{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_whatsnew_42_a' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-firstrun-learnmore-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{# No need to display up to date messaging on firstrun #}
+{% block up_to_date_message %}{% endblock %}
+
+{% block main_subhead %}
+  {# TODO need to update this link to fall back to the Private Browsing product page #}
+  <a class="button-flat-dark" href="https://support.mozilla.org/kb/private-browsing-use-firefox-without-history/" data-highlight="privateWindow" role="button">{{ _('Try the new Private Browsing') }}</a>
```

Comment:
I think we're good to merge this one and then follow up with another PR once the landing page is merged. I don't think it's really worth holding up?